lib/tests/streamlit/elements/data_editor_test.py

```diff
@@ -431,6 +431,8 @@ def test_with_compatible_data(
             DataFormat.PYSPARK_OBJECT,
             DataFormat.PANDAS_INDEX,
             DataFormat.PANDAS_STYLER,
+            DataFormat.MODIN_OBJECT,
+            DataFormat.SNOWPANDAS_OBJECT,
```

Comment:
nit: could you add it to the list in an alphabetical order please to make it easier for humans to read what is tested? For example so that all `SNOW*` formats are grouped together